github/Organization.py

```diff
@@ -577,9 +582,18 @@ def create_repo(
             post_parameters["allow_rebase_merge"] = allow_rebase_merge
         if delete_branch_on_merge is not github.GithubObject.NotSet:
             post_parameters["delete_branch_on_merge"] = delete_branch_on_merge
+        if visibility is not github.GithubObject.NotSet:
+            post_parameters["visibility"] = visibility
```

Comment:
Thinking about it, we may also want assert on the content of visibility, such as this example from git grep:

```
github/Repository.py:        assert state in ["open", "closed"], state
```